src/components/errorbars/attributes.js

```diff
@@ -89,48 +96,57 @@ module.exports = {
         valType: 'integer',
         min: 0,
         dflt: 0,
-        role: 'info'
+        role: 'info',
+        editType: 'dostyle'
     },
     tracerefminus: {
         valType: 'integer',
         min: 0,
         dflt: 0,
-        role: 'info'
+        role: 'info',
+        editType: 'dostyle'
     },
     copy_ystyle: {
```

Comment:
Should we :hocho: these in v2?

Reply:
(I'd vote _yes_)

Reply:
`traceref` or `copy_ystyle`? The former absolutely 🔪  but I kinda like the latter.

Reply:
I was referring to `copy_ystyle` and `copy_zstyle`, but I don't mind keeping them in v2 :smile:

Reply:
yeah, they're kind of ugly, but it's such a common use case to want the same style for both/all directions (and in fact I'd say such an *uncommon* use case to have *different* styles) that I think they deserve to stay.